src/dlgAboutDialog.cpp

```diff
@@ -1062,3 +1072,17 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
     textBrowser_supporters->setDocument(supportersDocument.get());
     textBrowser_supporters->setOpenExternalLinks(true);
 }
+
+QString dlgAboutDialog::createBuildInfo() const {
+
+    QString commitHash = !QByteArray(GIT_COMMIT_HASH).isEmpty() ? QStringLiteral(GIT_COMMIT_HASH) : QUESTION_MARK;
+    QString branch = !QByteArray(GIT_BRANCH).isEmpty() ? QStringLiteral(GIT_BRANCH) : QUESTION_MARK;
+    return QStringLiteral("<table border=\"0\" style=\"margin-bottom:36px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n")
+        .append(QStringLiteral("<tr><td colspan=\"2\" style=\"font-style:italic;\">%1<br></td></tr>").arg(tr("It would be useful to copy and include information below when reporting an issue!")))
```

Comment:
I don't think this is adding value, we should remove this sentence. There's a lot of things we can say are useful in Mudlet, we're not going to mark all of them!

Reply:
Well, I put it in as a hint as what we wanted when I suggested we removed the *Copy to clipboard* button - as it would be difficult (AFAICT) to put a button *within* the HTML (or even the whole widget) to do that...

Reply:
Actually probably not that hard :) Got little idea that would make it possible actually.